FOSSAsia/MapViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MapViewController.swift
+//  FOSSAsia
+//
+//  Created by SIMRANPREET KAUR on 06/06/18.
+//  Copyright © 2018 FossAsia. All rights reserved.
+//
+
+import Foundation
+import MapKit
+import CoreLocation
+
+class MapViewController: UIViewController {
+
+    @IBOutlet weak var mapView: MKMapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let locationCoordinates = CLLocationCoordinate2DMake(Constants.Location.lattitude, Constants.Location.longitude)
```

Comment:
Why is the location coordinate hard-coded? I think we are getting location coordinate from server side.

Reply:
@imjog I have added map to show location for fossAsia office only. I am studying the backend api part of it!!